lib/spack/spack/error.py

```diff
@@ -134,7 +137,19 @@ def __init__(self, provided, required=None, constraint_type=None, conflicts=None
             indented = ['  %s\n' % conflict for conflict in conflicts]
             conflict_msg = ''.join(indented)
             msg = '%s is unsatisfiable, conflicts are:\n%s' % (provided, conflict_msg)
+
+            newline_indent = '\n    '
+            if not debug:
+                msg += newline_indent
+                msg += 'To see full clingo unsat cores, re-run with `spack -d`'
+            if not debug or not minimize_cores:
+                msg += newline_indent
+                msg += 'For minimal full cores, re-run with `spack -d --minimize-cores'
```

Comment:
```suggestion
                msg += 'For minimal full cores, re-run with `spack -d --minimize-full-cores'
```

Reply:
Changing it in the other place instead, thanks for catching that!